ts/parts/PieSeries.ts

```diff
@@ -821,15 +821,17 @@ seriesType<Highcharts.PieSeriesOptions>(
          * @private
          */
         getX: function (
-            this: Highcharts.VariablePieSeries,
+            this: Highcharts.PieSeries,
             y: number,
             left: boolean,
             point: Highcharts.PiePoint
         ): number {
             var center = this.center,
                 // Variable pie has individual radius
-                radius = this.radii ?
-                    this.radii[point.index as any] :
+                radius = (this as Highcharts.VariablePieSeries).radii ?
```

Comment:
Shouldn't this be `PieSeries` too? `PieSeries` is base series, `VariablePieSeries` is created on top of `PieSeries`

Reply:
PieSeries has no radii property.

Reply:
This happens, if you do no proper class inheritance with function masking and instead modify the base function.

Reply:
Ah, I see now. Thanks for the explanation!